.github/workflows/opensearch-sink-opendistro-integration-tests.yml

```diff
@@ -6,7 +6,13 @@ name: Data Prepper OpenSearchSink integration tests with Open Distro
 on:
   push:
     branches: [ main ]
+    paths:
+      - 'data-prepper-plugins/opensearch/**'
+      - '*gradle*'
```

Comment:
Does this path mean it runs when the root `gradle.build` file is modified?

Reply:
Yes, it should cover both the `gradle.build` and the `settings.gradle`.

Per GitHub documentation:
> `*`: Matches zero or more characters, but does not match the `/` character. 

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet